src/lib/time-formats.js

```diff
@@ -1,14 +1,50 @@
-/* exported formats */
-
-export const formats = {
-  default: {
-    short_time: '[@] ha',
-    short_date_time: 'ddd [@] ha',
-    long_date_time: 'ddd MMM D \ [@] ha'
-  },
-  'it': {
-    short_time: '[@] H',
-    short_date_time: 'ddd [@] H',
-    long_date_time: 'ddd D MMM \ [@] H'
+/* exported getLocalizedDateTime */
+
+const uiLocale = [browser.i18n.getUILanguage().replace('_', '-'), 'en-US'];
```

Comment:
`uiLocales` maybe to indicate that it's a language fallback chain, not a single locale?